packages/libs/user-datasets/src/lib/Components/Detail/UserDatasetDetail.jsx

```diff
@@ -552,11 +501,81 @@ class UserDatasetDetail extends React.Component {
         <div style={{ maxWidth: '600px' }}>
           <Mesa state={compatibilityTableState} />
         </div>
-        {compatibilityInfo}
+        {compatibilityStatus}
       </section>
     );
   }
 
+  getCompatibilityStatus() {
+    const { userDataset, config, dataNoun } = this.props;
+    const { projectId } = config;
+
+    const { status, projects } = userDataset;
+
+    /**
+     * In VDI, we know a dataset is compatible when the site-specific's install status
+     * indicates a successful install.
+     *
+     * We know a dataset is incompatible when the site-specific's install status
+     * indicates `missing-dependency`
+     */
+    const installStatusForCurrentProject = status.install?.find(
+      (d) => d.projectId === projectId
+    );
+
+    const isInstallable = projects.includes(projectId);
```

Comment:
Maybe we can change this variable name to something like `isTargettingCurrentSite`? `isInstallable` could be interpreted to mean that there are errors preventing installation.